### ISSUE TITLE
Commit fa3dc39 - can't create directory '/encfs/raw/user/user-*'

### ISSUE THREAD

**Issue by scriptzteam** [hackerschoice/segfault#28]
Hi, and thx much for fixes :)

now it is build up, but another errors spawned. i can ssh onto 0.0.0.0:2222 but its just sit there on "Creating Server blaBlaBla........................" then just exits the ssh, seems its due to that it can not create that user folder

It has something todo with those lines probably:
https://github.com/hackerschoice/segfault/blob/67a64bcc535e2654c959bf720c2dbf88fd6da768/encfsd/encfsd.sh#L125
https://github.com/hackerschoice/segfault/blob/67a64bcc535e2654c959bf720c2dbf88fd6da768/encfsd/encfsd.sh#L126

After i do:
```SF_BASEDIR="$(pwd)" SF_SSH_PORT=2222 sfbin/sf up```

I get:
```sf-encfsd_1           | mkdir: can't create directory '/encfs/raw/user/user-YzVlNGEzMm': No such file or directory```

Also running it second time after CTRL+C bring this one:
```[2022-10-17 13:31:03] [WARN] Not found: data/share/GeoLite2-City.mmdb
Starting segfault_router-fix-network_1 ... done
Starting segfault_sf-redis_1           ... done
Starting segfault_dns-doh_1            ... done
Starting sf-router                     ... done
Starting segfault_sf-encfsd_1          ... done
Starting segfault_sf-portd_1           ... done
Starting segfault_dnsmasq_1            ... done
Starting segfault_sf-destructor_1      ...
Starting segfault_nginx_1              ... error
Starting sf-host                       ...
Starting segfault_gsnc_1               ...
Starting segfault_sf-destructor_1      ... done
Starting sf-mullvad                    ...
Starting sf-cryptostorm                ...
Starting segfault_gsnc_1               ... done
Starting sf-nordvpn                    ... done
Starting sf-mullvad                    ... done
Starting sf-cryptostorm                ... done

ERROR: for nginx  Cannot start service nginx: error while creating mount source path '/dev/shm/sf/encfs-sec/www-root': mkdir /dev/shm/sf/encfs-sec/www-root: file exists

ERROR: for segfault  Cannot start service segfault: error while creating mount source path '/dev/shm/sf/encfs-sec/www-root': mkdir /dev/shm/sf/encfs-sec/www-root: file exists
ERROR: Encountered errors while bringing up the project.
```

**Comment by scriptzteam** [comment]
Temporary fix?

https://github.com/hackerschoice/segfault/pull/39

**Comment by SkyperTHC** [comment]
fixed in 0.3.7a. Thank you for reporting.
